ivadomed/scripts/visualize_and_compare_testing_models.py

```diff
@@ -161,9 +159,12 @@ def visualize_and_compare_models(ofolders, metric="dice_class0", metadata=None):
 
             folders = [Path(folder).resolve().name] * len(scores)
             subject_id = result["image_id"]
-            combined = np.column_stack((folders, scores.astype(np.object, folders), subject_id)).T
-            singleFolderDF = pd.DataFrame(combined, columnNames).T
```

Comment:
Note that this double-transpose (`.T.T`) was unnecessary (the two cancelled each other out), so I excluded them from the rewrite here.